index.js

```diff
@@ -69,6 +69,9 @@ if (program.saveSequence) {
       for (var output in step.info.outputs) {
         console.log("[" + program.step + "]: " + output + " = " + step[output]);
       }
+    },
+    onNotify: function(msg) {
+      console.log('\x1b[36m%s\x1b[0m','🌟  '+msg);
```

Comment:
Does this overwrite the default in /src/ui/...? Just checking!

Reply:
Not much though providing a beautiful version in console.log